### DISCUSSION TITLE
essay/2024/01/22/chatgpt-price

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#167] [Comments]
# essay/2024/01/22/chatgpt-price

ChatGPT를 구독해서 쓰고 있습니다. 월 20달러. 약 27,000원. 넷플릭스보다 비쌉니다.

https://jeho.page/essay/2024/01/22/chatgpt-price.html

**Comment by websocketbear** [comment]
저도 예전에는 검색해서 알아봤던 대부분을 이제는 먼저 GPT에 물어보고 진행합니다. 심지어, 에러가 난 코드가 있으면 그거 조금 마사지하고 복사&붙여넣기를 하여 물어보기도 합니다. 정말 자주 사용하고 있어요. 구독료가 비싸지만 그만한 값어치를 하는 거 같아요. 주변의 개발자들은 생각보다 안쓰는 것 같은데, 저는 GPT 덕분에 주변 사람들에게 물어보는게 많이 줄었습니다.
저도 똑똑하다는 생각이 전혀 들지 않기 때문에 그들의 틈바구니에서 살아 남기 위해 열심히 사용하고 있어요!

**Reply by BenjaminKim** [reply]
저랑 똑같이 사용하고 계신데요?😁
주변 사람들이 안 써주면… 오히려 좋은 현실입니다.(웃음)

**Comment by wonkyDD** [comment]
맞아요 정말 돈값하더라고요 질문자체가 즐거워진 거 같아요..
이전에는 모르는게 나오면 북마크해두고 todo에 쌓아뒀던 것과 비교하면 정말 즐거워요 😀

키워드 얻기도 편하고 잘 대답못해도 기반으로  검색해 관련된 블로그들도 찾기쉬워서 행복합니다.
항상 글 잘 보고 있습니다.

**Reply by BenjaminKim** [reply]
@wonkyDD 저도 언젠간 검색해봐야지 하고 todo에만 넣어두고 찾아보지 않은 질문들이 정말 많아요.
10년 동안 궁금해만 하고 모르고 지낸 것들이 무지 많았어요. 늦게 나마 배워가면서 너무 좋습니다!
